src/main/kotlin/org/rust/lang/core/stubs/index/RsModulesIndex.kt

```diff
@@ -63,7 +64,11 @@ class RsModulesIndex : StringStubIndexExtension<RsModDeclItem>() {
         private fun key(mod: RsModDeclItem): String? {
             val pathAttribute = mod.pathAttribute
             return if (pathAttribute != null) {
-                FileUtil.getNameWithoutExtension(PathUtil.getFileName(pathAttribute))
+                val fileName = PathUtil.getFileName(pathAttribute)
+                if (fileName == RsConstants.MOD_RS_FILE)
```

Comment:
This works but I wonder if is there a better way to extract the module name from a string like "foo1/foo2/mod.rs"

Reply:
No, I think this is fine!